src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+/* stylelint-disable selector-list-comma-newline-after */
+@import url(https://fonts.googleapis.com/css2?family=Caveat&family=Roboto:wght@500&display=swap);
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  width: 1200px;
+  height: 60px;
+  margin: auto;
+  padding-top: 30px;
+}
+
+.nav_link , .is-active {
+  flex-direction: row;
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 7.5px;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+  border-bottom: 4px solid #00acdc;
```

Comment:
Move all repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)